cmd/ebrelayer/txs/relayToEthereum.go

```diff
@@ -20,11 +20,9 @@ import (
 
 const (
 	// GasLimit the gas limit in Gwei used for transactions sent with TransactOpts
-	GasLimit            = uint64(500000)
+	GasLimit = uint64(6721975)
```

Comment:
1.8 million gas should be fine for this file.

Reply:
6 million is excessive

Reply:
Yes, I maxed it during development and never reduced it.  Pushed a change.